### PR TITLE
remove mali-onerror and mali-compose

### PR DESCRIPTION
**Pull request by bojand**
The two new packages provide typings in the module themselves now.

Removed using `not-needed` npm script.

### CODE REVIEW

**Review by danwbyrne — APPROVED** [review]

File: notNeededPackages.json [hunk 1]
```diff
@@ -936,6 +936,18 @@
             "sourceRepoURL": "https://github.com/malijs/mali",
             "asOfVersion": "0.9.2"
         },
+        {
+            "libraryName": "mali-compose",
+            "typingsPackageName": "mali-compose",
+            "sourceRepoURL": "https://github.com/malijs/mali-compose",
+            "asOfVersion": "0.1.2"
```
Comment (PranavSenthilnathan) [inline notNeededPackages.json]: The latest version published is [1.1.0](https://www.npmjs.com/package/@types/mali-compose). Please update the version here to a value higher than the published version.
```suggestion
            "asOfVersion": "1.2.0"
```

**Review by PranavSenthilnathan — CHANGES_REQUESTED** [review]

**Comment by bojand** [comment]
1.2.0 is the new version that bundles the type definition. Would that not be the correct value to use for `asOfVersion`?

**Comment by PranavSenthilnathan** [comment]
Yes, that would be fine, but the version in the current PR is 0.1.2, not 1.2.0.

**Comment by bojand** [comment]
Yes I see. Sorry... I was reading this on my phone and must have misread the "suggested change" from GitHub as my actual diff. Not sure how I got the version incorrect in the PR at the start. Thanks for noticing! Should be addressed now.

**Comment by orta** [comment]
Blast from the past, I've rebased and removed the merge conflicts and it came to this - do you still want it merged?

**Comment by weswigham** [comment]
`notNeededPackages.json` already has an entry for this (referring to the more up-to-date namespaced package), so this shyouldn't be needed anymore.

**Pull request closed without merging**